src/client/sandbox/event/drag-and-drop/data-transfer-item-list.js

```diff
@@ -0,0 +1,149 @@
+import DataTransferItem from './data-transfer-item';
+import DATA_STORE_MODE from './data-store-mode';
+import DATA_TRANSFER_ITEM_KIND from './data-transfer-item-kind';
+
+function parseTextUriList (textUriList) {
```

Comment:
Move in to the `DataTransferItemList` as internal, non-enumerable function `_parseTextUriList`